packages/listbox/types/ListboxMixinTypes.d.ts

```diff
@@ -51,10 +51,16 @@ export declare class ListboxHost {
 
   protected _listboxReceivesNoFocus: boolean;
 
+  protected _noTypeAhead: boolean;
+
   protected _uncheckChildren(): void;
 
   private __setupListboxNode(): void;
 
+  protected _handleTypeAhead(ev: KeyboardEvent, { setAsChecked: boolean }): void;
+
+  private __clearTypeAheadTimeout: Function | undefined;
```

Comment:
hmm, forgot this one. Should be __pendingTypeAheadTimeout wih type number|undefined